lib/components/timer.dart

```diff
@@ -35,7 +35,7 @@ class _CountdownTimerPageState extends State<CountdownTimerPage>
         CountdownTimerController(endTime: endTime, onEnd: onEnd, vsync: this);
   }
 
-  void onEnd() async {
+  void onEnd() {
```

Comment:
You reverted the changes made in recent commits, kindly undo those

Reply:
Resolved!